common/resource/fx.go

```diff
@@ -41,6 +41,7 @@ import (
 	"go.temporal.io/server/common/searchattribute"
 	"go.temporal.io/server/common/telemetry"
 	"go.temporal.io/server/common/testing/testhooks"
+	"go.temporal.io/server/service/history/configs"
```

Comment:
This is in `common`, shared by all services. It shouldn't have a dependency on history service.